test/WebJobs.Script.Tests/EndToEndTimeoutTests.cs

```diff
@@ -0,0 +1,229 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Diagnostics;
+using System.Linq;
+using System.Runtime.ExceptionServices;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs.Host;
+using Microsoft.Azure.WebJobs.Host.Timers;
+using Xunit;
+
+namespace Microsoft.Azure.WebJobs.Script.Tests
+{
+    public class EndToEndTimeoutTests
+    {
+        [Fact]
+        public async Task TimeoutTest_SyncFunction_Node()
```

Comment:
That is my goal but it felt nicer to keep all timeout tests on their own -- they don't use the Fixture in those other tests and they have a bunch of specific startup/shutdown code.

---
In reply to: [76700341](https://github.com/Azure/azure-webjobs-sdk-script/pull/604#discussion_r76700341) [](ancestors = 76700341)